packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -322,9 +323,10 @@ export class OdspDocumentStorageService extends OdspDocumentStorageServiceBase {
 						} else {
 							// Note: There's a race condition here - another caller may come past the undefined check
 							// while the first caller is awaiting later async code in this block.
-
+							cacheLookupTimeInSerialFetch = performance.now();
```

Comment:
>cacheLookupTimeInSerialFetch [](http://example.com/codeflow?start=7&length=28)

shouldnt' this have a different variable ?

Reply:
renamed